### DISCUSSION TITLE
Advice for training style with dreambooth using a large dataset

### DISCUSSION THREAD

**Discussion by InushaDeSilva** [d8ahazard/sd_dreambooth_extension#1091] [Q&A]
I have a dataset of 11,000 images and I wish to train a style using these images. I'm wondering what settings and number of regulization images I should use for this

For now I was thinking of using
- 45 steps per image
- learning rate of 8e-7
- 25,000 regulization images
- constant learning rate with a starting factor of 0.9

Any advice on this would be greatly appreciated

yes, I know the training would take forever with this much data but I'm not sure if decreasing the trading steps would affect my training quality

**Comment by alelordelo** [comment]
Hi, Im also curious about DB on large datasets, but didn't find any info around...

 Did you ran the training?
